src/components/ChatLog.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import ChatEntry from './ChatEntry';
+
+const ChatLog = (props) => {
+    const { entries, onLikeChange } = props;
+    const handleLikeChange = (id, liked) => {
+        onLikeChange(liked);
+    };
```

Comment:
Instead of adding this here, it would be best to just pass down the function as-is and have the ChatEntry function just use the `liked` argument instead of using both `id` and `liked` since the original function does not need the `id`.